features/user/components/userProfile/friendsPanel/FriendsListItem.tsx

```diff
@@ -0,0 +1,143 @@
+'use client'
+
+import { Avatar, AvatarFallback, AvatarImage } from '@/components/ui/avatar'
+import { Button } from '@/components/ui/button'
+import { useToast } from '@/components/ui/use-toast'
+import { createInvitation, deleteFriend } from '@/features/user/actions'
+import { User } from '@/features/user/types/User'
+import { Invitation } from '@prisma/client'
+
+import React, { useState, useTransition } from 'react'
+import { FaTrash, FaUser, FaUserClock, FaCheck, FaPlus } from 'react-icons/fa'
+
+interface FriendsListItemProps {
+	user: User
+	variant: 'friends' | 'add'
+	currentUserId: string
+	invitations?: Invitation[]
+	onRefreshUsers?: () => void
+	filter?: string
+}
+
+export const FriendsListItem = ({
+	user: { name, image, id },
```

Comment:
to jest wysyłający zaproszenie? Jeżeli tak to słaba nazwa, powinno być sendingUser czy coś takiego bo user to można pomyśleć że aktualnie zalogowany user, ale chyba nie o niego chodzi skoro przekazujesz również propa currentUserId